exercises/variable-length-quantity/tests/variable-length-quantity.rs

```diff
@@ -0,0 +1,89 @@
+extern crate variable_length_quantity as vlq;
+
+#[test]
+fn to_single_byte() {
+    assert_eq!(&[0x40], vlq::to_bytes(&[0x40]).as_slice());
+    assert_eq!(&[0x7f], vlq::to_bytes(&[0x7f]).as_slice());
```

Comment:
Is there a value to having both of these tests? Is there an implementation detail that's tested in the second assert that is not tested in the first assert?